src/controller/bankrupt/bankrupt-officer.controller.ts

```diff
@@ -5,10 +5,10 @@ import { logger, formattingOfficersInfo } from '../../utils';
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   try {
     const bankruptOfficer = await fetchBankruptOfficer(req.params?.id);
-
-    if(!bankruptOfficer.error){
+    
+    if (!bankruptOfficer.error) {
       const officer = (bankruptOfficer.data) ? formattingOfficersInfo([bankruptOfficer.data])[0] : {};
-      return res.render('bankrupt_officer', { bankruptOfficer: officer });
+      return res.render('bankrupt_officer', { bankruptOfficer: officer } );
     } else if( bankruptOfficer.status === 500) {
       return res.status(bankruptOfficer.status).render('error-pages/500');
     } else {
```

Comment:
I would not have committed a file just with spaces changes :)

Reply:
this was trying to revert previous changes to match main, mainly to avoid merge issues